internal/nightfallconfig/nightfall_config.go

```diff
@@ -81,8 +83,8 @@ func GetNightfallConfigFile(workspacePath, fileName string, logger logger.Logger
 	if err != nil {
 		return nil, err
 	}
-	if len(nightfallConfig.Conditions) < 1 {
-		return nil, errors.New("Nightfall config file is missing Conditions")
+	if nightfallConfig.ConditionSetUUID == "" && len(nightfallConfig.Conditions) < 1 {
```

Comment:
Is `ConditionSetUUID` and `Conditions` mutually exclusive? I seem to remember that the call to DSG would fail if we send a request with both `ConditionSetUUID` and `Conditions` set.

Reply:
Yup that's correct. This is handled in the lambda (if we have both when we hit the lambda, the lambda will default to using the `ConditionSetUUID` in the request to DSG).